status-im/status-react
# feat(wallet): disable complex routing

fixes #20869 
fixes #20895

### Summary

This PR disables routes that involves more than 1 transactions. If best prop of the suggested routes response contains more than 1 element, we replace it with an empty vector.

Linked issues are different, but now that we are disabling multichain routes, expected behavior would be the same for both issues.

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

- Restore a user with available assets on at least 3 networks.
- Try to build a multi-route.
- Expected message should be "No routes found"

status: ready

### Risk

Described potential risks and worst case scenarios.

Tick **one**:
